recipes/systemd.rb

```diff
@@ -1,41 +1,48 @@
 include_recipe '::dd-agent'
 
 # Monitor Systemd
-#
-# node.default['datadog']['systemd']['instances'] = [
-#   {
-#     'unit_names' => [
-#       'myservice1.service',
-#       'myservice2.service',
-#       'mysocket.socket'
-#     ],
-#     'substate_status_mapping' => [
-#       'services' => [
-#         'myservice1' => {
-#           'running' => 'ok',
-#           'exited' => 'critical'
-#         },
-#         'myservice2' => {
-#           'plugged' => 'ok',
-#           'mounted' => 'ok',
-#           'running' => 'ok',
-#           'exited' => 'critical',
-#           'stopped' => 'critical'
-#         }
-#       ],
-#       'sockets' => [
-#         'mysocket' => {
-#           'running' => 'ok',
-#           'exited' => 'critical'
-#         }
-#       ]
-#     ],
-#     'tags' => [
-#       'mykey1:myvalue1',
-#       'mykey2:myvalue2'
-#     ]
-#   }
-# ]
+# It works for all unit types like targets, services, sockets, devices, mounts, automounts, swaps, paths, timers, snapshots, slices, scopes, busnames
+node.default['datadog']['systemd']['instances'] = [
+  {
+    'unit_names' => [
+      'myservice1.service',
+      'myservice2.service',
+      'mysocket.socket',
+      'mytimer.timer'
+    ],
+    'substate_status_mapping' => [
+      'services' => [
+        'myservice1' => {
+          'running' => 'ok',
+          'exited' => 'critical'
+        },
+        'myservice2' => {
+          'plugged' => 'ok',
+          'mounted' => 'ok',
+          'running' => 'ok',
+          'exited' => 'critical',
+          'stopped' => 'critical'
+        }
+      ],
+      'sockets' => [
+        'mysocket' => {
+          'running' => 'ok',
+          'exited' => 'critical'
+        }
+      ],
+      'timers' => [
+        'mytimer' => {
+         'running' => 'ok',
+         'exited' => 'critical'
+        }
+      ]
+    ],
+    'tags' => [
+      'mykey1:myvalue1',
+      'mykey2:myvalue2'
+    ]
+  }
+]
```

Comment:
I think this example should remain in a comment, to be in line with how other integrations are defined.

Reply:
Hi @KSerrania, yeah you are right. I simply forgot to comment that in. Will adjust the PR. Thank you for the hint!